docs/topics/forms/descriptions/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Input and description
+layout: default
+parent: Web forms
+nav_order: 4
+---
+
+# Input field and description
+
+A description provides extra information about how fill out a form field. 
+
+In this section you find information about how to connect one or more descriptions to form fields and fieldsets. And also about the content, length and location for descriptions. 
```

Comment:
Grammar: 

"In this section you will find information about how to connect one or more descriptions to form fields and fieldsets. You will also learn best practices for the content, length and location of descriptions.